app/src/debug/kotlin/codes/chrishorner/socketweather/data/DebugNetworkComponents.kt

```diff
@@ -50,42 +46,30 @@ class DebugNetworkComponents(
   moshi: Moshi
 ) : NetworkComponents {
 
-  @Deprecated("Delete once migration to Compose is complete.")
-  val debugRetrofitConfig: DebugRetrofitConfig
-  @Deprecated("Delete once migration to Compose is complete.")
-  val httpLogger = HttpLogger(app, prettyPrintJson = true)
-
-  private val httpLogger2 = HttpLoggingInterceptor(object : Logger {
-    override fun log(message: String) {
-      val formattedMessage: String = try {
-        when {
-          message.startsWith('{') -> JSONObject(message).toString(2)
-          message.startsWith('[') -> JSONArray(message).toString(2)
-          else -> message
-        }
-      } catch (e: JSONException) {
-        message
+  private val logger = HttpLoggingInterceptor { message ->
+    val formattedMessage: String = try {
+      when {
+        message.startsWith('{') -> JSONObject(message).toString(2)
+        message.startsWith('[') -> JSONArray(message).toString(2)
```

Comment:
😂 is whitespace technically a valid start character? *looks up JSON spec*

Reply:
Seems to be the case 😭 may want to trim leading and trailing whitespace

Reply:
Not worth the effort for debug logs on an API that should never return a result beginning with whitespace.